### PR TITLE
Added Haskell helloworld

### PR DESCRIPTION
**Pull request by mattfaltyn**

### CODE REVIEW

File: exercises.md [hunk 1]
```diff
@@ -1,7 +1,10 @@
 # Exercises
 
+<<<<<<< HEAD
+=======
```
Comment (nkohen) [inline exercises.md]: Looks like this still has a couple artifacts from a rebase

**Review by nkohen — COMMENTED** [review]
probably need to add a `.gitignore` in the `haskell` directory so that the non-source stuff doesn't get added to git (i.e. `*.exe` and `*.o` and the like) (probably want to `git rm` the ones that got through as well once that has happened)

**Comment by mattfaltyn** [comment]
> probably need to add a `.gitignore` in the `haskell` directory so that the non-source stuff doesn't get added to git (i.e. `*.exe` and `*.o` and the like) (probably want to `git rm` the ones that got through as well once that has happened)

Good point - will make the changes now

File: exercises.md (hunk 1 above)
Comment (mattfaltyn) [reply exercises.md]: Thanks for the heads up - I forgot to pull the most recent commit to my local branch.

**Comment by nkohen** [comment]
Should be good to go once you delete the conflict scars :)

**Comment by nkohen** [comment]
Replaced by #10

**Pull request closed without merging**